code/game/objects/items/storage/boxes.dm

```diff
@@ -41,6 +41,9 @@
 /obj/item/storage/box/ComponentInitialize()
 	. = ..()
 	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	STR.storage_flags = STORAGE_FLAGS_VOLUME_DEFAULT
+	STR.max_volume = STORAGE_VOLUME_CONTAINER_S
```

Comment:
It's a bit small. I'd give it its own value `STORAGE_VOLUME_BOX` and maybe make it 6 total small items?

Reply:
but then it's bigger than the space it takes up...

Reply:
i rescind this

Reply:
One thing that can be thought about with boxes is it could be easier to keep things organized in a backpack.

It's a lot easier to pack more things in a suitcase or a hiking pack when it's in containers that help things fit orderly.

Boxes have always held more than they take up, but not too much more.